### PR TITLE
October 26th Release Notes

### PR DESCRIPTION
**Pull request by orangejulius**
Look at all these nice changes :)

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
+* Asking the `/reverse` endpoint to return venues from Geonames used to return a error message. However, it should work, and now [it does](https://github.com/pelias/api/pull/1045)!
+
+### Code changes
+
+* Our [Placeholder](https://github.com/pelias/placeholder/pull/59) and [interpolation](https://github.com/pelias/interpolation/pull/84) services now have access logs. This should make it much easier to diagnose any issues and measure traffic
+* The Point in Polygon service now can download data using the [Who's on First downloader](https://github.com/pelias/pip-service/pull/37) directly. Its a bit easier to set up the PIP service as a result!
+* You can now specify the network interface the Pelias API will [listen on](https://github.com/pelias/api/pull/1031), by setting the `HOST` environment variable. The default is still Node.js's default to listen on all interfaces.
+* Its now easy to tell what parser was used for a query, as the API will set `parser: "libpostal"` when [libpostal was used](https://github.com/pelias/api/pull/1017)
```
Comment (trescube) [inline release-notes.md]: Its -> It's

Comment (trescube) [reply release-notes.md]: "what parser" -> "which parser"

File: release-notes.md [hunk 2]
```diff
@@ -4,6 +4,20 @@
...
+
+### Bug fixes
+
+* The `/place` endpoint would sometimes return records from a [different layer than the one you asked for](https://github.com/pelias/api/pull/1036), but no longer!
+* Asking the `/reverse` endpoint to return venues from Geonames used to return a error message. However, it should work, and now [it does](https://github.com/pelias/api/pull/1045)!
+
+### Code changes
+
+* Our [Placeholder](https://github.com/pelias/placeholder/pull/59) and [interpolation](https://github.com/pelias/interpolation/pull/84) services now have access logs. This should make it much easier to diagnose any issues and measure traffic
+* The Point in Polygon service now can download data using the [Who's on First downloader](https://github.com/pelias/pip-service/pull/37) directly. Its a bit easier to set up the PIP service as a result!
```
Comment (trescube) [inline release-notes.md]: Its -> It's

File: release-notes.md [hunk 3]
```diff
@@ -4,6 +4,20 @@
 > code changes as well as new features. There are also [release notes](https://mapzen.com/documentation/search/release-notes/)
 > specifically for the hosted instance of Pelias run by Mapzen, Mapzen Search.
 
+## 26 October 2017
+
+### Bug fixes
+
+* The `/place` endpoint would sometimes return records from a [different layer than the one you asked for](https://github.com/pelias/api/pull/1036), but no longer!
+* Asking the `/reverse` endpoint to return venues from Geonames used to return a error message. However, it should work, and now [it does](https://github.com/pelias/api/pull/1045)!
```
Comment (dianashk) [inline release-notes.md]: _an_ error

File: release-notes.md (hunk 1 above)
Comment (orangejulius) [reply release-notes.md]: come on we obviously have an unlimited number of parsers to choose from

**Comment by orangejulius** [comment]
All typos/grammar fixed. thanks for the suggestions :)

**Pull request merged**